data/tooltips.txt

```diff
@@ -774,7 +774,7 @@ tip "% firing shields / second:"
 	`Relative shields loss when this weapon is fired, converted to loss per second to allow easy comparisons between different weapons.`
 
 tip "shots / second:"
-	`How many projectiles per second this weapon fires. Weapons that fire continuously do not require batteries if the ship's energy generation is higher than the weapon's energy consumption.`
+	`How many projectiles per second this weapon fires. Weapons that fire continuously do not require batteries if the ship's energy generation is higher than the weapon's energy consumption. Except weapons which fire continuously, but only in bursts. For these weapons, the percentage uptime will be given in brackets.`
```

Comment:
It seems odd to me to describe how continuous fire weapons behavior, then give an exception, but not describe how that exception should be handled.

How about this instead?
```suggestion
	`How many projectiles per second this weapon fires. Weapons that fire continuously do not require batteries if the ship's energy generation is higher than the weapon's energy consumption. Some continuous fire weapons only fire a fraction of the time, in which case the ship's energy production must be higher than the weapon's energy consumption divided by the firing time.`
```

So if a weapon has a firing energy / sec of 100 and shows a firing time of 50%, you'd need 100/50% = 200 energy generation in order to not need a battery.